kani-driver/src/harness_runner.rs

```diff
@@ -70,6 +71,33 @@ impl<'sess, 'pr> HarnessRunner<'sess, 'pr> {
 
         Ok(results)
     }
+
+    /// Return an error if the user is trying to verify a harness with stubs without enabling the
+    /// experimental feature.
+    fn check_stubbing(&self, harnesses: &[&HarnessMetadata]) -> Result<()> {
+        if !self.sess.args.is_stubbing_enabled() {
+            let with_stubs: Vec<_> = harnesses
+                .iter()
+                .filter_map(|harness| {
+                    (!harness.attributes.stubs.is_empty()).then_some(harness.pretty_name.as_str())
+                })
+                .collect();
+            match with_stubs.as_slice() {
+                [] => { /* do nothing */ }
+                [harness] => bail!(
+                    "Harness `{}` contains stubs which are unstable.\n\
+                    To enable stubbing, pass options `-Z stubbing`",
+                    harness
+                ),
+                harnesses => bail!(
+                    "Harness `{}` contains stubs which are unstable.\n\
```

Comment:
"Harnesses" and "contain"

Reply:
Also I'm generally a bit unhappy with the phrasing "stubs which are unstable", it sounds as if *these particular* stubs are unstable, not stubbing in general.